src/test/java/com/talentpath/airTNB/services/ActivityTests.java

```diff
@@ -0,0 +1,138 @@
+    }
+
+    @Test
+    @Transactional
+    void getAllActivities(){
+        List<Activity> activities = activityRepo.findAll();
+        assertEquals(1, activities.size());
+
+        Activity politicsTour = new Activity();
+        politicsTour.setCity("Washington DC");
+        politicsTour.setPrice(75);
+        activityRepo.saveAndFlush(politicsTour);
+
+        Activity tourFacebook = new Activity();
+        tourFacebook.setCity("Menlo Park");
+        tourFacebook.setPrice(30);
+        activityRepo.saveAndFlush(tourFacebook);
+
+        activities = activityRepo.findAll();
+        assertEquals(3, activities.size());
+        assertEquals("Washington DC", activities.get(1).getCity());
+        assertEquals(75, activities.get(1).getPrice());
+        assertEquals("Menlo Park", activities.get(2).getCity());
+        assertEquals(30, activities.get(2).getPrice());
+    }
+
+    @Test
+    @Transactional
+    void getActivityById(){
+        List<Activity> activities = activityRepo.findAll();
+        assertEquals(1,activities.get(0).getId());
+
+        Activity activity1 = new Activity();
+        activityRepo.saveAndFlush(activity1);
+        Activity activity2 = new Activity();
+        activityRepo.saveAndFlush(activity2);
+
+        activities = activityRepo.findAll();
+        assertEquals(2,activities.get(1).getId());
+        assertEquals(3,activities.get(2).getId());
+    }
+
+    @Test
+    @Transactional
+    void getActivityByCity(){
+        Activity activity2 = new Activity();
+        activity2.setCity("Houston");
+        activity2.setPrice(22);
+        activityRepo.saveAndFlush(activity2);
+
+        Activity activity3 = new Activity();
+        activity3.setCity("Houston");
+        activity3.setPrice(33);
+        activityRepo.saveAndFlush(activity3);
+
+        Activity activity4 = new Activity();
+        activity4.setCity("Houston");
+        activity4.setPrice(44);
+        activityRepo.saveAndFlush(activity4);
+
+        Activity activity5 = new Activity();
+        activity5.setCity("San Francisco");
+        activity5.setPrice(99);
+        activityRepo.saveAndFlush(activity5);
+
+        Activity activity6 = new Activity();
+        activity6.setCity("San Francisco");
+        activity6.setPrice(909);
+        activityRepo.saveAndFlush(activity6);
+
+        Activity activity7 = new Activity();
+        activity7.setCity("Salt Lake City");
+        activity7.setPrice(15);
+        activityRepo.saveAndFlush(activity7);
+
+        Activity activity8 = new Activity();
+        activity8.setCity("Sacramento");
+        activity8.setPrice(77);
+        activityRepo.saveAndFlush(activity8);
+
+        Activity activity9 = new Activity();
+        activity9.setCity("Sacramento");
+        activity9.setPrice(88);
+        activityRepo.saveAndFlush(activity9);
+
+        List<Activity> activitiesInHouston = activityRepo.findByCity("Houston");
+
+        assertEquals(4,activitiesInHouston.size());
```

Comment:
should assert on the fact that all of the items in this list have the city "Houston", not their price. e.x.

`activitiesInHouston.stream().forEach(activity -> assertEquals("Houston",activity.getCity())`

Reply:
the same applies for salt lake, sf and sacremento

Reply:
I was just testing the price to make sure it was the correct object within the list.  I was trying to understand what you meant by using the stream for it but I guess that is a much more efficient method in testing the city for the entire list.  I was just doing some manual testing to ensure that it was the correct object but I like the idea of adding in a stream & lambda for something more scalable

Reply:
yup! it makes sense why you were testing those. The philosophy I'm approaching these tests with is that they need to make sure that they're testing the property specific to the test name.

Reply:
Thinking about it a little bit more, I want to amend my "not" in the original comment. Its definitely a good idea to test more properties of the object in addition to making sure that each activity has the correct city

Reply:
Just went ahead and updated & added the changes